draft-irtf-cfrg-hpke.md

```diff
@@ -420,34 +420,23 @@ and {{encryption-context}} for details.
 Suppose we are given a KDF, and a Diffie-Hellman group providing the
 following operations:
 
-- `GenerateKeyPair()`: Randomized algorithm to generate a key pair `(skX, pkX)`
-  for the DH group in use
-- `DeriveKeyPair(ikm)`: Deterministic algorithm to derive a key pair
-  `(skX, pkX)` from the byte string `ikm`, where `ikm` SHOULD have at
-  least `Nsk` bytes of entropy (see {{derive-key-pair}} for discussion)
-- `DH(skX, pkY)`: Perform a non-interactive DH exchange using the
-  private key `skX` and public key `pkY` to produce a Diffie-Hellman
-  shared secret of length `Ndh`. This function can raise a
-  `ValidationError` as described in {{validation}}.
-- `Serialize(pk)`: Produce a byte string of length `Npk`
-  encoding the public key `pk`
-- `Deserialize(enc)`: Parse a byte string of length `Npk` to recover a
-  public key. This function can raise a `DeserializeError` error upon
-  `enc` deserialization failure.
+- `DH(skX, pkY)`: Perform a non-interactive Diffie-Hellman exchange using
```

Comment:
Why don't we need the removed methods any more?  Is this just to avoid pass-through?

Reply:
Yeah, these methods are already on the KEM interface, and the text clarifies that these functions produce keys in the group.